pytest.ini

```diff
@@ -79,6 +79,7 @@ markers =
     sno: SingleNodeOpenShift tests
     infrastructure: Infrastructure tests
     data_protection: Data Protection tests
+    cross_cluster_live_migration: Cross Cluster Live Migration tests
```

Comment:
should be under `Configuration requirements`

Reply:
added the requirement for the remote cluster 
but I think it should be a separate lane, not part of any other lane